src/template.json

```diff
@@ -20,8 +21,10 @@
         "SourceAmiName": "{{ .SourceAMIName }}"
       },
       "type": "amazon-ebs",
-      "user_data_file": "./bootstrap_win.txt",
-      "winrm_password": "SuperS3cr3t!!!!",
```

Comment:
I read in the packer doc you linked in the description "You'll notice that this config does not define a user or password; instead, Packer will ask AWS to provide a random password that it generates automatically." Which sounds like an improvement, but how do you get access to the password then? Does it show up in the log when packer creates the image? What's the process for keeping track of it then, put in LP?

Reply:
OK.  Just to be clear there are two groups of users in windows so let's discuss them separately.

Administrator - you get the admin password by provisioning an instance with this AMI then you goto EC2-> select the instance -> Actions ->Connect -> provide the imagecentral ssh private key (get it from lastpass) -> click "Get Password" button.  Now you can connect to it with Admin user.

End User - provision an instance with this AMI thru the SC and we have things in the SC that associate the provisioned instance with jumpcloud.  The under user uses their JC credentials to login.

Reply:
👍

Reply:
you can also connect to the instance through SSM via your IAM user, in which case it will not need the Admin user's password... you can set it to whatever value you want after connecting too.

Reply:
Ahh cool! @ahayden is correct, I verified that you can also connect with Actions -> Connect -> Session Manager